lib/widgets/adaptive_flat_button.dart

```diff
@@ -0,0 +1,31 @@
+import 'dart:io';
+
+import 'package:flutter/material.dart';
+import 'package:flutter/cupertino.dart';
+
+class AdaptiveFlatButton extends StatelessWidget {
+  final String text;
+  final Function handler;
+
+  AdaptiveFlatButton(this.text, this.handler);
+
+  @override
+  Widget build(BuildContext context) {
+    return Platform.isIOS
+                        ? CupertinoButton(
+                            child: Text(
+                              text,
+                              style: TextStyle(fontWeight: FontWeight.bold),
+                            ),
+                            onPressed: handler,
+                          )
+                        : FlatButton(
+                            textColor: Theme.of(context).primaryColor,
+                            child: Text(
+                              text,
+                              style: TextStyle(fontWeight: FontWeight.bold),
+                            ),
+                            onPressed: handler,
+                          );
+  }
+}
```

Comment:
add new line at the end of file

Reply:
unresolving this. see this comment: https://github.com/MapleSyyrup/Personal-Expenses/pull/12#discussion_r533463164